templates/taxi/car_detail.html

```diff
@@ -18,5 +18,16 @@ <h1>Drivers</h1>
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
+  {% if user in car.drivers.all %}
+  <form method="post" action="{% url 'car-remove-driver' car.pk user.pk %}">
```

Comment:
The URL name 'car-remove-driver' used in the form action might not be defined in your `urls.py`. Ensure that this URL pattern is correctly defined to handle the removal of a driver from a car.